src/scripts.js

```diff
@@ -133,7 +136,8 @@ function showFavoritesPage() {
 }
 
 function viewRecipe(event) {
-  selectedRecipeName = event.target.innerText;
+  selectedRecipeName = event.target.innerText
+  console.log(selectedRecipeName)
```

Comment:
need to remove console.log - fine for now but just to remember for future pushes